webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -60,7 +63,7 @@ const componentRegistry = {
   },
 };
 
-const coreComponets = [
+export const coreComponets = [
```

Comment:
I don't think we should export the `coreComponents`. I think you can work with the `componentRegistry` instead.

```jsx
const Component = componentRegistry.getComponent(name);
```